x-pack/plugins/session_view/public/components/detail_panel_host_tab/index.tsx

```diff
@@ -5,194 +5,411 @@
-            description: (
-              <DetailPanelCopy
-                textToCopy={`host.os.family: "${hostData.os.family}"`}
-                tooltipContent={hostData.os.family}
-              >
-                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
-                  {hostData.os.family}
-                </EuiTextColor>
-              </DetailPanelCopy>
-            ),
-          },
-          {
-            title: <DetailPanelListItem>os.full</DetailPanelListItem>,
-            description: (
-              <DetailPanelCopy
-                textToCopy={`host.os.full: "${hostData.os.full}"`}
-                tooltipContent={hostData.os.full}
-              >
-                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
-                  {hostData.os.full}
-                </EuiTextColor>
-              </DetailPanelCopy>
-            ),
-          },
-          {
-            title: <DetailPanelListItem>os.kernel</DetailPanelListItem>,
-            description: (
-              <DetailPanelCopy
-                textToCopy={`host.os.kernel: "${hostData.os.kernel}"`}
-                tooltipContent={hostData.os.kernel}
-              >
-                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
-                  {hostData.os.kernel}
-                </EuiTextColor>
-              </DetailPanelCopy>
-            ),
-          },
-          {
-            title: <DetailPanelListItem>os.name</DetailPanelListItem>,
-            description: (
-              <DetailPanelCopy
-                textToCopy={`host.os.name: "${hostData.os.name}"`}
-                tooltipContent={hostData.os.name}
-              >
-                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
-                  {hostData.os.name}
-                </EuiTextColor>
-              </DetailPanelCopy>
-            ),
-          },
-          {
-            title: <DetailPanelListItem>os.platform</DetailPanelListItem>,
-            description: (
-              <DetailPanelCopy
-                textToCopy={`host.os.platform: "${hostData.os.platform}"`}
-                tooltipContent={hostData.os.platform}
-              >
-                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
-                  {hostData.os.platform}
-                </EuiTextColor>
-              </DetailPanelCopy>
-            ),
-          },
-          {
-            title: <DetailPanelListItem>os.version</DetailPanelListItem>,
-            description: (
-              <DetailPanelCopy
-                textToCopy={`host.os.version: "${hostData.os.version}"`}
-                tooltipContent={hostData.os.version}
-              >
-                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
-                  {hostData.os.version}
-                </EuiTextColor>
-              </DetailPanelCopy>
-            ),
-          },
-        ]}
-      />
+      <EuiAccordion
```

Comment:
I think you can use the `DetailPanelAccordion` component here, if you add `initialIsOpen` as a prop there. so that we can remove some of the similar code here

Reply:
For the nested accordion, you could try adding an optional `children` prop to `DetailPanelAccordion` and use it for the nested accordion
```
<DetailPanelAccordion title="..." listItems={[...]}>
  {nestedAccordion}
</DetailPanelAccordion>
```